### PR TITLE
Install AWS requirements in the test environment.

### PR DESCRIPTION
**Pull request by skschneider**
AWS disk integration tests actually provision AWS VMs.

### CODE REVIEW

File: tests/requirements_test.py [hunk 1]
```diff
@@ -25,6 +25,11 @@
 
 _PATH = 'path'
 
+_FAKE_PACKAGE_REQUIREMENTS = """
```
Comment (noahl) [inline tests/requirements_test.py]: Did you mean to include this file in this PR?

**Comment by noahl** [comment]
I think you might have included an extra file accidentally. I'm not sure about this change - if you need `requirements-aws.txt` to use tox, why not just include the AWS deps in `requirements-testing.txt`? Or if you don't need them for testing, then why are they in the tox testing section?

I think we might need a separate tox environment for integration tests. That way the regular testing environment won't be affected by extra requirements for integration tests.

**Comment by skschneider** [comment]
Separate integration test environment: #868 and #869.
If those changes are more favorable, I will close this PR.

**Comment by skschneider** [comment]
Closing in favor of #868 and #869.

**Pull request closed without merging**